### DISCUSSION TITLE
Is there any way to make a branch deployment in dagster open source

### DISCUSSION THREAD

**Discussion by lahdirakram** [dagster-io/dagster#19382] [Q&A]
hello, im trying to make deployments according to Git branches i see that it is possible via dagster cloud, but is there any way to do something similar with open-source?

**Comment by prha** [comment]
No, branch deployments are only available in Dagster Cloud.

**Comment by geoHeil** [comment]
hm you could deploy multiple instances of dagster using some automation like pulumi and CI
